lib/models/builder.js

```diff
@@ -171,7 +171,7 @@ class Builder extends CoreObject {
       .catch(error => {
         this.processAddonBuildSteps('buildError', error);
         throw error;
-      });
+      }).finally(() => this.project.configCache.clear());
```

Comment:
Minor nit-pick, the rest of this promise chain is dot aligned, I would expect this `.finally` to be on the next line.